### DISCUSSION TITLE
Fix for Drop Down Liste at Buttom

### DISCUSSION THREAD

**Discussion by Impedanz** [home-assistant/frontend#18967] [Dashboard]
If I want to select an entity, trigger, etc. in an automation via a drop-down, it can happen when I am at the bottom of the page that not all entities can be displayed because the drop-down list is cut off at the bottom. So I have to create an IF THEN in an automation, for example, and expand it so that there is enough space at the bottom to be able to see the complete list above it. Is a fix possible?

Translated with DeepL.com (free version)
![Screenshot 2023-12-08 130106](https://github.com/home-assistant/frontend/assets/63118609/40dd8257-e9c5-4966-91cc-3e35ab10799f)
![Screenshot 2023-12-08 130046](https://github.com/home-assistant/frontend/assets/63118609/a881562d-3b8a-47a7-a11c-5be4c9d47339)

**Comment by Quentame** [comment]
Should be an issue, not a discussion
